src/common/components/Actions.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Button, CardActions, Link } from '@mui/material';
+
+const Actions = ({ label, destination }) => {
```

Comment:
Issue: I don't think this component name is describing its intent. It seems to be a specific component to handle navigation links inside cards that only hold one action.

Suggestion: Make this just a customization of `Button` (Remove `CardActions` from this file) and improve the name related to the navigation intent

Reply:
How about `CardActionButton`? Is that better? If not, can you customize Button as you would like it?